tests/request.spec.js

```diff
@@ -37,5 +39,43 @@ describe('YotiResponse', () => {
           .catch(done);
       });
     });
+
+    context('when making an API request', () => {
+      const expectedHeaders = {
+        'X-Yoti-SDK': 'Node',
+        'X-Yoti-SDK-Version': yotiPackage.version,
+        'X-Yoti-Auth-Key': new RegExp('^[a-zA-Z0-9]+'),
```

Comment:
Since this is a public key, we can also put a length check on the RegEx.

Additionally I think we can with the digest.

Reply:
Updated to check length in 1a9f5626509663d674980c59ad2af2d8c169d936